src/app/client-master/assessment-year-dialog/assessment-year-dialog.component.ts

```diff
@@ -0,0 +1,52 @@
+import { Component, OnInit, Inject } from '@angular/core';
+import { MatDialogRef, MAT_DIALOG_DATA, ErrorStateMatcher } from '@angular/material';
+import { DeleteClientsDialogComponent } from '../delete-clients-dialog/delete-clients-dialog.component';
+import { Client } from 'src/app/model/Client';
+import { DataTransferService } from 'src/app/shared/data/data-transfer.service';
+import { Router } from '@angular/router';
+import { FormControl, FormGroupDirective, NgForm, Validators, FormGroup, FormBuilder } from '@angular/forms';
+
+@Component({
+  selector: 'app-assessment-year-dialog',
+  templateUrl: './assessment-year-dialog.component.html',
+  styleUrls: ['./assessment-year-dialog.component.css']
+})
+export class AssessmentYearDialogComponent implements OnInit {
+  private assessmentYears = [
+    {value: '2019-20', viewValue: '2019-20'},
```

Comment:
shouldn't we extract this from db ? 
Like if a user has 2 years if data, 2 Financial year will be in drop down

Reply:
should not be hard coded I think

Reply:
Yes I know. Will do that later